src/entities/session.entity.ts

```diff
@@ -42,4 +43,13 @@ export class SessionEntity extends BaseBloomEntity {
     cascade: true,
   })
   sessionUser: SessionUserEntity[];
+
+  @OneToMany(
+    () => SessionFeedbackEntity,
+    (sessionFeedbackEntity) => sessionFeedbackEntity.session,
+    {
+      cascade: true,
+    },
+  )
+  sessionFeedback: SessionFeedbackEntity[];
```

Comment:
what was the reason for this change?

Reply:
See many to one documentation:
https://orkhan.gitbook.io/typeorm/docs/many-to-one-one-to-many-relations

I think it wasn't setup quite right before. Let me know if you agree. Forgot to write notes about these small changes. Apologies!